src/COWFeeModule.sol

```diff
@@ -84,20 +82,21 @@ contract COWFeeModule {
             buyToken: IERC20(toToken),
             receiver: address(receiver),
             sellAmount: 0,
-            buyAmount: 1,
+            buyAmount: 0,
             validTo: nextValidTo(),
             appData: appData,
             feeAmount: 0,
-            kind: SELL_KIND_HASH,
+            kind: GPv2Order.KIND_SELL,
             partiallyFillable: true,
-            sellTokenBalance: ERC20_BALANCE_HASH,
-            buyTokenBalance: ERC20_BALANCE_HASH
+            sellTokenBalance: GPv2Order.BALANCE_ERC20,
+            buyTokenBalance: GPv2Order.BALANCE_ERC20
         });
 
         for (uint256 i = 0; i < dripInteractions.length;) {
             SwapToken calldata swapToken = _swapTokens[i];
             order.sellToken = IERC20(swapToken.token);
             order.sellAmount = swapToken.sellAmount;
+            order.buyAmount = swapToken.buyAmount;
```

Comment:
I would suggest here that there is some checking that if the `swapToken.buyAmount == 0`, then set `order.buyAmount = 1`, otherwise this will result in invalid pre signatures being placed.

Reply:
will add this to the script